### PR TITLE
react(core): Introduce LinkWithClipboard component

### PR DESCRIPTION
**Pull request by caseyhebebrand**
A repeated code pattern in angular is the use of a link with a clipboard next to it. This component turns that code pattern into a react component and implements it on the instance details panel.

Screen Shot 2020-02-06 at 5 46 00 PM

### CODE REVIEW

File: app/scripts/modules/core/src/presentation/index.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export * from './hooks';
 export * from './HoverablePopover';
 export * from './LabelComponent';
 export * from './labeledValues';
+export * from './linkWithClipboard.component';
```
Comment (vigneshm) [inline app/scripts/modules/core/src/presentation/index.ts]: Should we make the react component available as well?

File: app/scripts/modules/core/src/presentation/LinkWithClipboard.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { CopyToClipboard } from 'core/utils';
+
+export interface ILinkWithClipboardProps {
+  url: string;
+  text: string;
+}
+
+export const LinkWithClipboard = ({ url, text }: ILinkWithClipboardProps) => (
+  <>
+    <a href={url} target="_blank">
```
Comment (vigneshm) [inline app/scripts/modules/core/src/presentation/LinkWithClipboard.tsx]: Would `target="_blank"` be a reasonable expectation for users or should we make it configurable?

File: app/scripts/modules/core/src/presentation/index.ts (hunk 1 above)
Comment (caseyhebebrand) [reply app/scripts/modules/core/src/presentation/index.ts]: Good call, I will add that for future use!

File: app/scripts/modules/core/src/presentation/LinkWithClipboard.tsx (hunk 2 above)
Comment (caseyhebebrand) [reply app/scripts/modules/core/src/presentation/LinkWithClipboard.tsx]: I think this is a reasonable expectation for users. This component will be used mainly for external links, in which case we would want a new page.

**Pull request merged**